core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -997,6 +1026,10 @@ private void onPostPlayback(final Playable playable, boolean ended, boolean skip
             media.onPlaybackPause(getApplicationContext());
         }
 
+        if (playable.hasAutoSkippedEnding()) {
```

Comment:
Hmm this adds quite a lot of code for a single use-case. What about checking the position here instead of adding the Playable methods?